github/config_service_test.go

```diff
@@ -74,7 +77,8 @@ func TestConfigService_TomlSave(t *testing.T) {
   host = "github.com"
   user = "jingweno"
   access_token = "123"
-  protocol = "https"`
+  protocol = "https"
+  unix_socket = "/tmp/go.sock"`
```

Comment:
These tests also congfigure `unix_socket`. It would be good if some tests didn't have this configuration, so they could keep exercising old logic.

Reply:
Split into separated tests in 0eb76ff